site.yml

```diff
@@ -47,3 +47,6 @@
     - role: fairfood
       become: yes
       become_user: "{{ app_user }}"
+
+    - role: nginx
+      become: yes
```

Comment:
Just a few thoughts here:

- The `become` doesn't need to be in both files.
- Ansible has the concept of *handlers* for this kind of action. They can be triggered by certain actions and are not executed if that action isn't run.
- We already have the role jdauphant.nginx and having two may be confusing.

Reply:
Good pickups. If I figure out what the problem really was, I could probably remove this altogether.